e2e_test/bin/e2e_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:io';
+
+import 'package:http/http.dart';
+import 'package:sentry/sentry.dart';
+
+const _exampleDsn =
+    'https://8b83cb94764f4701bee40028c2f29e72@o447951.ingest.sentry.io/5428562';
+
+const _org = 'sdk';
+const _projectSlug = 'dart';
+
+const _token = String.fromEnvironment('SENTRY_AUTH_TOKEN');
```

Comment:
Is this token already configured?

Reply:
will do

Reply:
idk if its used, but its there already https://github.com/getsentry/sentry-dart/blob/39c0194291855aea240fe1b1068837a66126c52f/.github/workflows/flutter.yml#L80-L81
let me know if the value is either not working or already rotated, so I can change it.

Reply:
It seems that the token is out of date

Reply:
please try again

Reply:
wondering if `String.fromEnvironment` also works for env. vars or only dart-define flags

Reply:
It works for environment variables and dart defines

Reply:
Actually it doesn't :(

Reply:
yeah I had the same issue with https://github.com/getsentry/sentry-dart/pull/545

I'm wondering if our usage of `String.fromEnvironment` in the SDK actually works, otherwise, we have a bug